rocks-db/src/clients/asset_client.rs

```diff
@@ -221,8 +218,14 @@ impl Storage {
             .into_iter()
             .filter_map(|asset| {
                 asset
-                    .filter(|a| !a.metadata.is_empty())
-                    .map(|a| (a.url.clone(), a))
+                    .filter(|a| {
```

Comment:
Once again, we don't check the metadata object for emptiness. So we may end up with a hashmap containing a url and an empty value. Is it expected from now on?